test/requirements_aistore.txt

```diff
@@ -1,2 +1,2 @@
 pytest
-aistore > 0.9.1
+aistore == 0.9.2
```

Comment:
Is there any reason to pin a specific version of `aistore`? If it's a hard requirement, we might need to check the version of `aistore` in `aisio.py` to prevent users using the lower version of `aistore`

Reply:
Sounds good.